packages/gatsby/src/commands/develop.ts

```diff
@@ -401,12 +401,12 @@ module.exports = async (program: IProgram): Promise<void> => {
       )
     }
 
-    program.ssl = await getSslCert({
+    program.ssl = (await getSslCert({
       name: sslHost,
       certFile: program[`cert-file`],
       keyFile: program[`key-file`],
       directory: program.directory,
-    })
+    })) as ICert
```

Comment:
I don't think this should be casted (surely TS can unbox the promise and type properly with a syntax `await`?). 

Additionally, the return value can be `false` here so the cast seems insufficient to me.

Reply:
fixed